src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -113,8 +111,8 @@ trait Backend {
    */
   def bindCall(workflowDescriptor: WorkflowDescriptor,
                key: CallKey,
-               locallyQualifiedInputs: CallInputs,
-               abortRegistrationFunction: AbortRegistrationFunction): BackendCall
+               locallyQualifiedInputs: CallInputs = Map.empty[String, WdlValue],
+               abortRegistrationFunction: AbortRegistrationFunction = AbortRegistrationFunction(_ => ())): BackendCall
```

Comment:
Maybe this should be an `Option[AbortRegistrationFunction]` rather than defaulting to an `AbortRegistrationFunction` that silently does nothing.